index.html

```diff
@@ -13,24 +14,34 @@
   <h1 class="head">&lt Coding Quiz &gt</Coding></h1>
  </header>
  <main class="main">
-   <section class="card">
-    <h2 class="question">Question 1</h2>
-     <p>What does HTML stand for?</p>
-     <button type="button" class="answer-button">Show Answer</button>
+   <section class="card" data-js="card">
+    <h2 class="card__question">Question 1</h2>
+     <p>What is semantic HTML?</p>
+     <button type="button"  class="card__answer-button">Show Answer</button>
    </section>
 
    <section class="card">
-    <h2 class="question">Question 2</h2>
-    <p>What does CSS stand for?</p>
-    <button type="button" class="answer-button">Show Answer</button>
+    <h2 class="card__question">Question 2</h2>
+     <p>What does CSS stand for?</p>
+     <p class="card__answer" data-js="card__answer">Cascading Style Sheet</p>
+    <button type="button" class="card__answer-button" data-js="card__answer-button">Show Answer</button>
    </section>
 
    <section class="card">
-    <h2 class="question">Question 3</h2>
+    <h2 class="card__question">Question 3</h2>
     <p>What are variables in JS?</p>
-    <button type="button" class="answer-button">Show Answer</button>
+    <button type="button" class="card__answer-button">Show Answer</button>
    </section>
  </main>
+ <footer>
+    <nav>
+      <a href="#" class="nav">Home</a>
```

Comment:
The class name is strange, because it`s not a `nav` element. Better go with `nav__link` or something similar.